docker/mysql/start.bash

```diff
@@ -59,4 +74,10 @@ done
 # file using the trick https://unix.stackexchange.com/a/185370/325629.
 mkdir -p /work && touch /work/mysql-inited
 
-sleep infinity
+
+# c.f. https://stackoverflow.com/questions/2935183/bash-infinite-sleep-infinite-blocking for BusyBox
```

Comment:
The voted answer is `sleep infinity ` from the URL, can you please add more comment to explain why remove `sleep infinity`?

Reply:
To minimize the image size, we did not install the `bash`, so we can't use `sleep infinity` here.